src/persist/benches/snapshot.rs

```diff
@@ -53,7 +53,7 @@ where
         .collect();
 
     let mut i = new_fn(1).expect("creating index cannot fail");
-    let id = i.register("0");
+    let id = i.register("0").expect("registering a stream cannot fail");
```

Comment:
nit: `.expect("stream has not been previously deleted")` or something like `.expect("registration succeeds")` if you're worried about it rotting